index.html

```diff
@@ -0,0 +1,168 @@
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <div class="resume-btn"><a href="#">Get My Resume</a></div>
+        </div>
+        <div class="about-me-skils">
+            <div class="worklanguages">
+                <img src="./images/icon-languages.svg" alt="Image depicting languages">
+                <h2>LANGUAGES</h2>
```

Comment:
- [ ] Well done on your "About me" section! 👍 However, I noticed that you have used an `<h2>` tag for the section title, while using an `<h2>` tag for the "Languages" title within the same section. To ensure proper heading hierarchy and maintain consistency, I recommend using an `<h3>` tag for the "Languages" title. Having a clear and logical heading structure not only improves accessibility but also enhances the overall organization and readability of your content. Good luck with your updates.

**_Note: Consider the same requirement for the "FRAMEWORKS" and "SKILLS" titles in this section._**